test/e2e/tests/04-submission-response.js

```diff
@@ -0,0 +1,149 @@
+const request = require('request');
+
+const report = require('../report');
+const config = require('../../../config');
+
+// FIXME: This should come in from a config variable
+const url = 'http://localhost:' + config.port; // eslint-disable-line
+
+const common = require('../utils/common.js');
+
+// TODO(awong): Move this into a custom command or assertion that can be used with client.expect.element().
+function expectNavigateAwayFrom(client, urlSubstring) {
```

Comment:
This is getting copied into enough places that it's wroth pulling out into a utility file now.  Tagging @akainic to put on her radar.